bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -7,13 +7,12 @@ private func dummy() {
     _ = NSLocalizedString("%s block options", comment: "translators: %s: block title e.g: \"Paragraph\".")
     _ = NSLocalizedString("%s block. Empty", comment: "translators: accessibility text for the media block empty state. %s: media type")
     _ = NSLocalizedString("%s block. This block has invalid content", comment: "translators: accessibility text for blocks with invalid content. %d: localized block title")
-    _ = NSLocalizedString("Add a description", comment: "")
     _ = NSLocalizedString("Add a shortcode…", comment: "")
-    _ = NSLocalizedString("Add annotation", comment: "")
     _ = NSLocalizedString("ADD AUDIO", comment: "")
     _ = NSLocalizedString("Add Block After", comment: "")
     _ = NSLocalizedString("Add Block Before", comment: "")
     _ = NSLocalizedString("ADD BLOCK HERE", comment: "")
+    _ = NSLocalizedString("Add button text", comment: "")
```

Comment:
👋 @geriux , noticed you've marked this thread as resolved so, when you get the chance, please add a comment here with the relevant decision made regarding fixing or not fixing this Hound report in this PR, thanks!

Reply:
Of course!

Since we now have Hound checking these files we'll need to reach a decision regarding how to add these missing comments for current and future ones. Since these entries are generated and pulled from the Gutenberg code base. We'll have to go through all of these missing comments in Gutenberg and add the needed information, then generate the strings file again.

As this change is recent and the check is not required I think we could merge once this PR is approved and not block it for this newly added Hound warning.

Reply:
**Update**: There's a way to disable this specific warning, I'll create a separate PR to test it out =)